README.md

```diff
@@ -7,367 +7,598 @@
 [BuildStatusURL]: https://travis-ci.org/coderaiser/supertape "Build Status"
 [CoverageURL]: https://coveralls.io/github/coderaiser/supertape?branch=master
 [CoverageIMGURL]: https://coveralls.io/repos/coderaiser/supertape/badge.svg?branch=master&service=github
+[MITLicenseURL]: https://opensource.org/licenses/MIT
+[MITLicenseIMGURL]: https://img.shields.io/badge/License-MIT-yellow.svg
 
 [![supertape](https://asciinema.org/a/Cgc3rDOfZAeDnJSxzEYpPfBMY.svg)](https://asciinema.org/a/Cgc3rDOfZAeDnJSxzEYpPfBMY)
 
-[Tape](https://github.com/substack/tape)-inspired [TAP](https://testanything.org/)-compatible simplest high speed test runner with superpowers.
+[**Tape**](https://github.com/substack/tape)-inspired [`TAP`](https://testanything.org/)-compatible simplest high speed test runner with superpowers.
 
-📼**Supertape** written from scratch after messing a lot with **tape**, it willing to be compatible with it as much as possible.
-and has a couple differences. It contains:
+----
 
-- ability to work with [ESM Modules](https://nodejs.org/api/esm.html) (take a look at [mock-import](https://github.com/coderaiser/mock-import) for mocking and 🎩[ESCover](https://github.com/coderaiser/escover) for coverage).
-- shows colored diff when test not `equal` or not `deepEqual`;
-- produces deteiled stack traces for `async functions`;
-- as many [`only`](https://github.com/coderaiser/supertape#testonlyname-cb) as you wish;
-- ability to [extend](https://github.com/coderaiser/supertape/blob/master/README.md#testextendextensions);
-- smart timeouts for long running tests 🏃‍♂️(configured with `SUPERTAPE_TIMEOUT`);
-- more natural assertions: `expected, result` -> `result, expected`, for example:
+📼 **Supertape** is a fast, minimal test runner with the soul of **tape**. It's designed to be as compatible as possible with **tape** while still having some key improvements, such as:
 
-```js
-t.equal(error.message, 'hello world', `expected error.message to be 'hello world'`);
-```
+- the ability to work with [ESM Modules](https://nodejs.org/api/esm.html) (take a look at [mock-import](https://github.com/coderaiser/mock-import) for mocking and 🎩[ESCover](https://github.com/coderaiser/escover) for coverage)
+- a number of [built-in pretty output formatters](#formatting)
+- easy [configuration](#list-of-options)
+- the ability to [extend](#extending)
+- showing colored diff when using the [`t.equal()`](#tequalactual-any-expected-any-message-string-since-v102) and [`t.deepEqual()`](#tdeepequalactual-any-expected-any-message-string-since-v102) assertion operators
+- detailed stack traces for `async` functions
+- not needing to specify the number of assertions with [`t.plan()`](https://github.com/substack/tape#tplann)
```

Comment:
`t.plan` is not supported but smart timeouts it’s important information.

Reply:
Yes, this was included to highlight that `t.plan()` doesn't exist in Supertape

Reply:
```suggestion
- [`t.plan()`](https://github.com/substack/tape#tplann) not supported
```

Reply:
Moved to "doesn't contain" with https://github.com/tommy-mitchell/supertape/commit/37fee4e4dc51135d5f892d35caf613e27f51e94b